www/_posts/2017-10-09-tools-release.md

```diff
@@ -0,0 +1,118 @@
+* [CB-13308](https://issues.apache.org/jira/browse/CB-13308) fixed issues with restoring plugins and platforms while using `npm@5+`.
+* [CB-12787](https://issues.apache.org/jira/browse/CB-12787) Fix plugin installation with `--link` option.
+* [CB-13056](https://issues.apache.org/jira/browse/CB-13056) added deprecation notice for **WebOS**
+* [CB-13057](https://issues.apache.org/jira/browse/CB-13057) added deprecation warning for `cordova platform save`
+
+To update your cordova CLI:
+
+    npm install -g cordova@latest
+
+Please report any issues you find at [issues.cordova.io](http://issues.cordova.io/)!
+
+<!--more-->
+# Changes include:
+
+## cordova-lib
+
+* [CB-13303](https://issues.apache.org/jira/browse/CB-13303) added `--save_exact`, `--production` flags
+* [CB-13288](https://issues.apache.org/jira/browse/CB-13288) updated `index.js` and test to fix `cordova plugin search`
+* [CB-13206](https://issues.apache.org/jira/browse/CB-13206) fixed incorrect target being passed in to `plugin add` from `restore-util.js`
+* [CB-13145](https://issues.apache.org/jira/browse/CB-13145) added `variable-merge.js` to deal with `plugin.xml` variables for uninstall
+* [CB-12870](https://issues.apache.org/jira/browse/CB-12870) catch all use cases for `getPlatformApiFunction` and update tests accordingly
+* [CB-12944](https://issues.apache.org/jira/browse/CB-12944) Platform's spec is ignored in `config.xml` if `package.json` doesn't contain dependency for platform
+* [CB-12361](https://issues.apache.org/jira/browse/CB-12361) added new unit tests for plugin tests
+* [CB-13020](https://issues.apache.org/jira/browse/CB-13020) (plugman) install filters out `nohooks`
+* [CB-13056](https://issues.apache.org/jira/browse/CB-13056) added deprecation notice for **WebOS**
+* [CB-13057](https://issues.apache.org/jira/browse/CB-13057) added deprecation warning for `cordova platform save`
+* [CB-12361](https://issues.apache.org/jira/browse/CB-12361) added tests for `save.js` and rebased
+* [CB-12895](https://issues.apache.org/jira/browse/CB-12895) switched from `jshint` to `eslint`
+* [CB-12361](https://issues.apache.org/jira/browse/CB-12361) updated `addHelper` tests
+* [CB-11980](https://issues.apache.org/jira/browse/CB-11980) Update `README` to reflect new repos
+* [CB-6143](https://issues.apache.org/jira/browse/CB-6143) Change `plugman.emit()` to `events.emit()`
+* Reorganized unit test directory. Changes include: - consolidate `spec-cordova/` and `spec-plugman/` into a single `spec/` dir. - put `jasmine config` and helper modules in top-level spec dir. - changed `package.json` npm run scripts to reflect purposes of tasks. remove `npm run ci`. Updated `README` to reflect `package.json` npm run script changes. 
+* [CB-12361](https://issues.apache.org/jira/browse/CB-12361) added unit tests for `prepare.spec.js`
+* Update cordova-lib api. Deprecate `raw` from api calls. 
```

Comment:
Can we call this out more prominently for downstream tooling?

Reply:
agreed. Any thoughts on wording for this